bip-update-process.md

```diff
@@ -0,0 +1,706 @@
+providing a brief overview of BIP statuses (see [Workflow](#workflow) below) to the audience.
+
+## BIP format and structure
+
+### Specification
+
+Authors may choose to submit BIPs in MediaWiki or Markdown[^markdown] format.
+
+Each BIP must have a _Preamble_, an _Abstract_, and a _Copyright_ section. Authors should consider all issues in the
+following list and address each as appropriate. For some BIPs, some concerns may not warrant a dedicated section.
+
+* Preamble -- Headers containing metadata about the BIP (see the section [BIP header preamble](#bip-header-preamble)
+  below).
+* Abstract -- A short description of the technical issue being addressed.
+* Motivation -- The motivation is critical for BIPs. It should clearly explain what issue the BIP addresses, and how the
+  existing situation is inadequate to address the problem that the BIP solves.
+* Specification -- The technical specification should describe the syntax and semantics of any new feature. The
+  specification should be detailed enough to enable any Bitcoin project to create an interoperable implementation.
+* Rationale -- The rationale fleshes out the specification by describing what motivated the design and why particular
+  design decisions were made. It should describe related work and alternate designs that were considered. The rationale
+  should record relevant objections or important concerns that were raised and addressed as this proposal was developed.
+* Backward compatibility -- A section describing any backward incompatibilities, their severity, and instructions on how
+  implementers/users should deal with these incompatibilities.
+* Reference implementation -- Where applicable, a reference implementation, test vectors, and documentation must be
+  finished before the BIP can be given the status "Complete". Any or all of these can be provided either in the BIP or
+  as auxiliary files (see [Auxiliary Files](#auxiliary-files)).
+* Changelog -- A section to track modifications to a BIP after reaching Complete status.
+* Copyright -- The BIP must be placed under an acceptable license (see [BIP Licensing](#bip-licensing) below).
+
+#### BIP header preamble
+
+Each BIP must begin with an [RFC 822-style header preamble](https://www.w3.org/Protocols/rfc822/). The headers must
+appear in the following order. Headers marked with "\*" are optional. All other headers are required. The overview is
+followed by an explanation for each header.
+
+```
+  BIP: <BIP number, or "?" before assignment>
+* Layer: <Consensus (soft fork) | Consensus (hard fork) | Peer Services | API/RPC | Applications>
+  Title: <BIP title (≤ 50 characters)>
+  Authors: <Authors’ names and email addresses>
+* Shepherds: <Shepherds’ names and email addresses>
+  Status: <Draft | Complete | Deployed | Closed>
+  Type: <Specification | Informational | Process>
+  Created: <Date of number assignment, or "?" before assignment>
```

Comment:
It would be clearer to keep the date format of ISO 8601 (yyyy-mm-dd) as currently specified in BIP2.